templates/parachain/README.md

```diff
@@ -47,90 +70,143 @@ git clone https://github.com/paritytech/polkadot-sdk-parachain-template.git para
 cd parachain-template
 ```
 
-### Build
+## Starting a Development Chain
+
+### Omni Node
 
-🔨 Use the following command to build the node without launching it:
+[Omni Node](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/omni_node/index.html) can
+be used to run the parachain template's runtime. `polkadot-omni-node` binary crate usage is described at a high-level
+[on crates.io](https://crates.io/crates/polkadot-omni-node).
+
+#### Install `polkadot-omni-node`
+
+Please see the installation section at [`crates.io/omni-node`](https://crates.io/crates/polkadot-omni-node).
+
+#### Build `parachain-template-runtime`
 
 ```sh
-cargo build --release
+cargo build -p parachain-template-runtime --release
 ```
 
-🐳 Alternatively, build the docker image:
+#### Install `staging-chain-spec-builder`
+
+Please see the installation section at [`crates.io/staging-chain-spec-builder`](https://crates.io/crates/staging-chain-spec-builder).
+
+#### Use chain-spec-builder to generate the chain_spec.json file
 
 ```sh
-docker build . -t polkadot-sdk-parachain-template
+chain-spec-builder create --relay-chain "rococo-local" --para-id 1000 --runtime \
+    <target/release/wbuild/path/to/minimal-template-runtime.wasm> named-preset development
 ```
 
-### Local Development Chain
+**Note**: the `relay-chain` and `para-id` flags are mandatory information required by
+Omni Node, and for parachain template case the value for `para-id` must be set to `1000`, since this
+is also the value injected through [ParachainInfo](https://docs.rs/staging-parachain-info/0.17.0/staging_parachain_info/)
+pallet into the `parachain-template-runtime`'s storage.
```

Comment:
Not sure - it feels to be too detailed on this level. I'd remove it. Up to you.
```suggestion
```

Reply:
The concern is valid that the number 1000 is hardcoded in two places, and they have to be sync, and it is not obvious that they should. 

There should be some comment about this in this README, at least for now. 

How @iulianbarbu decides to do that I have less of an opinion on :) 

I would do this: 

1. make this README also be a docify-enabled one, and demonstrate where the number 1000 is repeated in the runtime.


Long term, we should make some kind of a node-RFC to move all of this info that is only causing annoyance to the runtime: 

1. Para-Id: should be defined in ONE place, and in the runtime.
2. Token symbol and decimals. 

@skunert would be happy to collaborate on this, and make it a 2025 project.